scripts/image_testing/chroot.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+if [ $(whoami) != "root" ] ; then
+  echo "Please run as root."
+  exit
+fi
+
+# perhaps this is the least commonly used util. perhaps not. 
+if [ "$(dpkg --get-selections qemu-user-static)" ] ; then
+  apt install binfmt-support qemu-user-static
+fi
+
+chroot root /bin/bash
```

Comment:
a case could be made here to set up bindmounts or instances of namespaced filesystems (/sys, /proc, /dev, etc), but there's a nonzero likelihood that a setup like this could damage the user's system; thus, with the loss of some functionality, we explicitly do not do this.